cas/service.c

```diff
@@ -1923,6 +1926,20 @@ static int cas_id_check(const struct m0_cas_id *cid)
 		layout = &cid->ci_layout;
 		if (layout->dl_type != DIX_LTYPE_DESCR)
 			rc = M0_ERR(-EPROTO);
```

Comment:
Here, replace this line with:
```
return M0_ERR(-EPROTO);
```
The checking of 'fom != NULL' is only applicable when the above checking is true.



Do the same for Line 1920.

Reply:
Good catch @huanghua78